dspace-rest/src/main/webapp/WEB-INF/security-applicationContext.xml

```diff
@@ -38,7 +38,7 @@
         <property name="authenticationManager" ref="dspaceAuthenticationManager"/>
         <!-- Ensures that after login no redirect is made to the home page, the rest will return a 200 status code -->
         <property name="authenticationSuccessHandler" ref="org.dspace.rest.authentication.NoRedirectAuthenticationLoginSuccessHandler"/>
-        <property name="usernameParameter" value="email"/>
+        <property name="usernameParameter" value="user"/>
```

Comment:
You are right. I've reverted the rename of the username parameter in this pull request.